### ISSUE TITLE
uninitialized constant CredentialsManager::AppfileConfig::Helper

### ISSUE THREAD

**Issue by slhodson969** [fastlane/fastlane#4225]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

I keep seeing `uninitialized constant CredentialsManager::AppfileConfig::Helper` in my console but don't understand why. I originally noticed it when running a lane but it happens at the most simple of times too, such as when checking versions of the installed gems.

```
scott$ fastlane -v
uninitialized constant CredentialsManager::AppfileConfig::Helper
uninitialized constant CredentialsManager::AppfileConfig::Helper
fastlane 1.81.0
```

```
scott$ sigh -v
uninitialized constant CredentialsManager::AppfileConfig::Helper
uninitialized constant CredentialsManager::AppfileConfig::Helper
uninitialized constant CredentialsManager::AppfileConfig::Helper
uninitialized constant CredentialsManager::AppfileConfig::Helper
uninitialized constant CredentialsManager::AppfileConfig::Helper
uninitialized constant CredentialsManager::AppfileConfig::Helper
uninitialized constant CredentialsManager::AppfileConfig::Helper
sigh 1.6.1
```

### Configuration Files

Appfile:
```ruby
 # The bundle identifier of your app
app_identifier "com.identifier"

# Your Apple email address
apple_id "scott@myemail.com"

# Developer Portal Team ID
team_id "[HIDDEN_TEAM_ID]"
```

### Environment

I'm fully up-to-date:

```
scott$ gem update fastlane credentials_manager
Updating installed gems
Nothing to update
```

**Comment by slhodson969** [comment]
Just wanted to add that, due to a company policy, I am not an admin on my laptop. Could that cause this? (it's caused me plenty of issues in the past)

**Comment by mfurtak** [comment]
Hi @slhodson969 - thanks for the report! :+1: 

This is a bug. :bug: Also, the symptom you're reporting is a bug.

Interestingly, the code path that is being triggered would imply that your `Appfile` contains so-called "smart quotes" (“ ” ‘ ’ ‛), which we're attempting to correct for you.

These can sneak in if the file gets modified with certain editors (e.g. TextEdit), so you may want to check on that!

As for the bug leading to the logging, I'll get that straightened out. Thanks!

**Comment by mfurtak** [comment]
We've merged #4228 as a fix for the erroneous logging. It will go out in the next release :rocket:

**Comment by mfurtak** [comment]
Hi @slhodson969 - This fix has gone out in [`credentials_manager` 0.15.1](https://github.com/fastlane/fastlane/releases/tag/credentials_manager/0.15.1) 🔑 ✨
